.java-version

```diff
@@ -0,0 +1 @@
+17
```

Comment:
Isn't it better to use the lowest supported version, so the issues are shown immediately?
What if someone doesn't have java 17 installed?

Reply:
thanks @pvary for your comment!

this matters only for people who use specific tooling to select java version for given project. `jenv` definitely reads those files but maybe others tools do too (would be reasonable).

So yes, if you have eg 8 installed and default, but 17 not installed, and you use jenv, then things won't work anymore, until you install 17.

If we put 8 here, and you have 17 installed and default but 8 not installed, , and you use jenv, then things won't work anymore, until you install 8.

While i agree that in general the lowest supported is the best, i would be weary putting 8 here, since it feels ancient and less likely to be already installed when someone checks the project out. This is for engineers convenience, so out could argue that newest stable & supported is the right choice for this file.